service/service.go

```diff
@@ -49,11 +50,17 @@ func (svc *Service) Init(ctx context.Context, cfg *config.Config, serviceList *E
 	// Get health client for api router
 	svc.routerHealthClient = serviceList.GetHealthClient("api-router", cfg.APIRouterURL)
 
+	dimensionClient, err := dimension.NewWithHealthClient(svc.routerHealthClient)
+	if err != nil {
+		return fmt.Errorf("failed to create dimensions API client: %w", err)
+	}
+
```

Comment:
You don't need this check here

Reply:
Think I do need this due to this: https://github.com/ONSdigital/dp-frontend-filter-flex-dataset/pull/12/files/a60227eec60733805d3e2795c957ce33693bdb35#r840434933